README.md

```diff
@@ -18,37 +18,45 @@ DOCKER_COMPOSE=docker-compose -f local-docker-compose.yml
 
 - install [Docker](http://docker.io)
 - bring in ICU source
-
-     cd src
-     git clone --branch master --depth 1 https://github.com/unicode-org/icu.git
-     cd icu
-     git lfs fetch
-     git lfs checkout
-     cd ../..
-
+```
```

Comment:
I think that putting the triple backticks at the start of the line breaks the list. Please try to indent each of these by two spaces.

Thanks for reducing the indent of the contents between the backticks. Don't change that.

Reply:
I think it's better now.